### PR TITLE
Initial Greek translation

### PR DESCRIPTION
**Pull request by Cruel**
From brigcaster on GBATemp. Needs review.

### CODE REVIEW

**Comment by brigcaster** [comment]
Hello, brigcaster from gbatemp here. There is an accidental omission in my translation. In greek the symbol of question mark is  **";"**  instead of **"?"**. This means that the lines 56, 395 must be corrected. I am not familiar with how github works, so I guess you are the only one that can make commits to the project, right?

**Comment by Cruel** [comment]
You can just add comments on the file if you want and I can change it.

In the future, you can just go to the freeShop repo (when the language is merged in) and make edits through Github and submit a PR. It can all be done through Github for convenience.

**Comment by brigcaster** [comment]
Alright, I will try it out now.

File: res/romfs/lang/greek.lang [hunk 1]
```diff
@@ -1,123 +1,123 @@
...
- Ok
+ Οκ
 
  Cancel
- Akyrwsi
+ Ακύρωση
 
 Are you sure you want\nto continue?
-Eiste sigoyros oti thelete na synexisete?
+Είστε σίγουρος ότι θέλετε να συνεχίσετε?
```
Comment (brigcaster) [inline res/romfs/lang/greek.lang]: ";" instead of "?"

File: res/romfs/lang/greek.lang [hunk 2]
```diff
@@ -126,270 +126,270 @@ Akyrwsi...
...
-Energopeiiste to kleisimo tis othonis meta apo apraksia
+Απενεργοποίηση οθόνης μετά από απραξία
 
 Remote title key URL(s)
-URL(s) apomakrismenwn title keys
+URL(s) απομακρυσμένων title keys
 
 You need to restart freeShop for\n%s to be loaded.\n\nWould you like to do this now?
-Prepei na epanakinisete to freeShop gia\nna fortwthei to %s.\n\nThelete na to kanete twra?
+Πρέπει να επανεκκινήσετε το freeShop\nγια να φορτωθεί το %s.\n\nΘέλετε να το κάνετε τώρα?
```
Comment (brigcaster) [inline res/romfs/lang/greek.lang]: ";" instead of "?"

**Comment by brigcaster** [comment]
Sorry, I didn't see the last question. The translation for "Installed" is "Εγκατεστημένο".

-

**Comment by Cruel** [comment]
Okay check it now.

And if you just check it on your own 3DS and make sure everything fits properly, then I can merge it.

**Comment by brigcaster** [comment]
I just tested it, the issue has been fixed. Did the portuguese translation have the same problem?

**Comment by Cruel** [comment]
The Portuguese translation was missing "Installed" so everyone copying it also missed it.

**Pull request merged**